mlem/contrib/kubernetes/base.py

```diff
@@ -0,0 +1,247 @@
+                    if event["object"].status.phase == "Running":
+                        w.stop()
+
+                deployments_list = (
+                    client.AppsV1Api().list_namespaced_deployment(
+                        namespace=f"mlem-{meta.image_name}-app"
+                    )
+                )
+
+                state.deployment_name = deployments_list.items[0].metadata.name
+                meta.update_state(state)
+
+            echo(
+                EMOJI_OK
+                + f"Deployment {state.deployment_name} is up in mlem-{meta.image_name}-app namespace"
+            )
+
+    def remove(self, meta: K8sDeployment):
+        self.check_type(meta)
+        with meta.lock_state():
+            meta.load_kube_config()
+            state: K8sDeploymentState = meta.get_state()
+            if state.deployment_name is not None:
+                client.AppsV1Api().delete_namespaced_deployment(
+                    name=state.deployment_name,
+                    namespace=f"mlem-{meta.image_name}-app",
+                )
+                client.CoreV1Api().delete_namespaced_service(
+                    name=state.deployment_name,
+                    namespace=f"mlem-{meta.image_name}-app",
+                )
+                client.CoreV1Api().delete_namespace(
+                    name=f"mlem-{meta.image_name}-app"
+                )
+                sleep(0.5)
+            echo(
+                EMOJI_OK
+                + f"Deployment {state.deployment_name} and the corresponding service are removed from mlem-{meta.image_name}-app namespace"
+            )
+            state.deployment_name = None
+            meta.update_state(state)
+
+    def get_status(
+        self, meta: K8sDeployment, raise_on_error=True
+    ) -> DeployStatus:
+        self.check_type(meta)
+        meta.load_kube_config()
+        state: K8sDeploymentState = meta.get_state()
+        if state.deployment_name is None:
+            return DeployStatus.NOT_DEPLOYED
+
+        pods_list = client.CoreV1Api().list_namespaced_pod(
+            f"mlem-{meta.image_name}-app"
+        )
+
+        return POD_STATE_MAPPING[pods_list.items[0].status.phase]
+
+
+class K8sYamlBuilder(MlemBuilder, K8sYamlBuildArgs):
+    type: ClassVar = "kubernetes"
+    image: DockerImage
+
+    def get_resource_args(self):
+        return K8sYamlBuildArgs(
+            image_name=self.image.name,
+            image_uri=self.image.uri,
+            image_pull_policy=self.image_pull_policy,
+            port=self.port,
+            service_type=self.service_type,
+        )
+
+    def build(self, obj: MlemModel):
+        resource_args = self.get_resource_args()
+
+        generator = K8sYamlGenerator(**resource_args.dict())
+        resource_yaml = generator.generate()
+
+        generator.write("resources.yaml")
+        echo(
+            EMOJI_OK
+            + f"resources.yaml generated for {obj.basename}, apply manualy using kubectl OR use mlem deploy"
```

Comment:
second part is probably redundant, at least we dont have anything like this in other builders

Reply:
removed `apply manualy using kubectl OR use mlem deploy`